tests/test_layout.py

```diff
@@ -12,8 +12,8 @@ class TestLayout:
     @pytest.mark.nondestructive
     @pytest.mark.xfail("'mozilla.com' in config.getvalue('base_url')",
```

Comment:
Not sure if it's the problem, but the condition doesn't need to be a string in recent versions of pytest. The following should work:

```python
@pytest.mark.skipif('mozilla.com' in pytest.config.getoption('base_url')
```

Until pytest-selenium 1.1 is released, it is correct that this will only apply to the command line option. In 1.1 it will also apply to a base URL from a config file.